app/views/govuk_component/analytics_meta_tags.raw.html.erb

```diff
@@ -1,17 +1,18 @@
 <%
-  links_hash = content_item.with_indifferent_access["links"] || {}
+
+  links_hash = content_item.to_h[:links] || {}
 
   organisations = []
-  organisations += links_hash["organisations"] || []
-  organisations += links_hash["lead_organisations"] || []
-  organisations += links_hash["supporting_organisations"] || []
-  organisations += links_hash["worldwide_organisations"] || []
+  organisations += links_hash[:organisations] || []
```

Comment:
`links_hash.fetch(:organisations, [])` would be more idiomatic. If you agree, you can apply it to all the `links_hash` accesses here.